hibernate-reactive-core/src/test/java/org/hibernate/reactive/schema/SchemaUpdateCockroachDBTestBase.java

```diff
@@ -44,7 +43,7 @@
 public abstract class SchemaUpdateCockroachDBTestBase extends BaseReactiveTest {
 
 	@Timeout(value = 10, timeUnit = MINUTES)
-
+	@EnableFor(COCKROACHDB)
```

Comment:
Just a curiosity, are this annotation inherited? Could I put them on the abstract class once instead?

Reply:
you are correct.  My previous annotations wouldn't allow it.